tests/test_kokkos.py

```diff
@@ -0,0 +1,73 @@
+# Copyright 2018-2022 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+Unit tests for Kokkos bindings.
+"""
+import pytest
+
+from pennylane_lightning_kokkos import InitArguments
+
+
+class TestKokkos:
+    """Tests that Kokkos bindings work."""
+
+    def test_InitArguments_init(self):
+        """Tests that InitArguments is properly initialized."""
+        args = InitArguments(1)
+        assert args.num_threads == 1
+        args = InitArguments(2)
+        assert args.num_threads == 2
+        args = InitArguments(5)
+        assert args.num_threads == 5
+
+    def test_InitArguments_init(self):
```

Comment:
This and the previous test have the same name.
Thoughts?

Reply:
Test parametrised too.